casepro/cases/views.py

```diff
@@ -426,7 +426,9 @@ def get_context_data(self, **kwargs):
         user = self.request.user
         partner = user.get_partner(org)
 
-        labels = Label.get_all(org, user).order_by('name')
+        labels = list(Label.get_all(org, user).order_by('name'))
+        Label.bulk_cache_initialize(labels)
```

Comment:
Open to suggestions on a better pattern for putting extra data into JSON model representations etc - data which isn't always needed, and is best fetched in bulk.